pep-8104.rst

```diff
@@ -0,0 +1,148 @@
+PEP: 8104
+Title: 2023 Term Steering Council election
+Author: Ee Durbin <ee@python.org>
+Sponsor: Brett Cannon <brett@python.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 08-Nov-2022
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the December
+2022 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2023 term
+(i.e. Python 3.12).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin, to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period shall be: November 14, 2022 through `November 28, 2022 AoE
+<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+nominations+close&iso=20221129T00>`_ [#note-aoe]_.
+
+The voting period shall be: December 1, 2022 through `December 14, 2022 AoE
```

Comment:
Its not *that* important since its the start date, not the end date. However, as mentioned previously this is potentially confusing, since AoE is only [defined](https://www.ieee802.org/16/aoe.html) and used for _deadlines_, not start times, and arguably the most logical extension of the concept to start times would imply a voting start time of 2022-11-30 10:00 UTC, rather than the 2022-12-01 12:00 UTC (26 hours later) that is listed below.

Reply:
It doesn't matter as folks can't vote until they get the email anyway, and once they do they can start. The start time is inconsequential as there's a physical blocker dictating when people can begin.